ruby/2.3/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM socrata/base
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get --force-yes -fuy install software-properties-common && \
+  DEBIAN_FRONTEND=noninteractive add-apt-repository -y ppa:brightbox/ruby-ng && \
+  DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get install -y ruby2.3 && \
+  DEBIAN_FRONTEND=noninteractive apt-get purge -y --auto-remove software-properties-common && \
+  rm -rf /var/lib/apt/lists/*
+
+# skip installing gem documentation
+RUN echo 'gem: --no-rdoc --no-ri --no-document' >> "/etc/gemrc" && \
+  gem install bundler
+
+# LABEL must be last for proper base image discoverability
+LABEL repository.socrata/ruby:2.3=""
```

Comment:
This should be as follows to match the name of the docker image (socrata/ruby2.3)
```dockerfile
LABEL repository.socrata/ruby2.3=""
```